src/components/organize/tasks/TaskList/TaskListItem.tsx

```diff
@@ -15,12 +14,12 @@ interface TaskListItemProps {
 
 
 const TaskListItem = ({ task, hrefBase }: TaskListItemProps): JSX.Element => {
-    const { id, title, published, deadline, expires } = task;
+    const { id, title } = task;
     const taskStatus = getTaskStatus(task);
 
     return (
         <Link
-            href={ hrefBase + `/tasks/${id}` }
+            href={ hrefBase + `/calendar/tasks/${id}` }
```

Comment:
The addition of `calendar` here seems to be the reason why the tabs are not working as expected. It does not match the `baseHref` in the layout.